Scripts/Runtime/Entities/PersistentData/PersistentDataManager.cs

```diff
@@ -0,0 +1,116 @@
+using Anvil.CSharp.Collections;
+using Anvil.Unity.DOTS.Jobs;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    public static class PersistentDataManager
```

Comment:
Should this be a static class? 

Its nice and easy to access, but then you also have to dispose it when stopping the game and going to a menu for example. 

If this was an object, I'm curious where to place it.

In my mind there are the following use cases:

- ThreadPersistentData 
    - This is global to the App. It shouldn't matter what world we're in because you can only access one thread at a time.
- EntityPersistentData 
    - This, I think, should be global to the World no? As it's written today, if you can have `Entity(3 : 2)` in WorldA and `Entity(3: 2)` in WorldB and they are different Entities then I think this could conflict.
- PersistentData 
    - This I also think should global to the World? But then we might also want one that it global to the app.

So maybe there needs to be a "Scope" value as well. (App or World)

In terms of placement, it could go on it's own System and we get the benefit of it being destroyed when the System gets destroyed. App scope is kind of messed up then but we could have the World Scope managers lazy get or create one singular App scope manager and redirect to that. When all World managers are destroyed, then the App one destroys itself too.

I don't like having to get the system though, but maybe we could do that by default in our base system. So the API then becomes:

```csharp
//PersistentData is World scoped to this System we are writing this line of code in. (Or TaskDriver)
PersistentData.CreatePersistentData(id, scope, etc)
```